gui/mainwindow.cc

```diff
@@ -510,8 +507,8 @@ void MainWindow::outputFileNameEdited()
 QString MainWindow::filterForFormat(int idx)
 {
   QString str = formatList_[idx].getDescription();
-  str.replace(QRegExp("\\("), "[");
-  str.replace(QRegExp("\\)"), "]");
+  str.replace('(', '[');
```

Comment:
This would be a good starter project for someone new to the project. Add a flag for -^4 or -^json or something that emitted something better defined than TSV that could be read by code like this.

Reply:
Agreed.  Where is our army of enthusiastic eager young volunteers?